app/assets/stylesheets/styles/components/users/components/_profile.css.scss

```diff
@@ -0,0 +1,167 @@
+.member-profile {
+
+  .canopy .canopy {
+    background-color: rgba(0, 0, 0, 0.15);
+
+    p, h2, h5 {
```

Comment:
Selector should have depth of applicability no greater than 3, but was 4<br>Each selector in a comma sequence should be on its own line